gef-ui/server/controllers/mocks/index.js

```diff
@@ -85,45 +86,50 @@ const MockApplicationResponseDraft = () => ({
   comments: [],
   ukefDealId: null,
   createdAt: chance.timestamp(),
-  submissionDate: chance.timestamp(),
 });
 
-const MockApplicationResponseSubmitted = () => ({
-  _id: '1234',
-  exporter: {},
-  bankId: 'BANKID',
-  bankInternalRefName: 'Internal reference',
-  additionalRefName: 'Additional reference',
-  status: CONSTANTS.DEAL_STATUS.UKEF_IN_PROGRESS,
-  userId: 'mock-user',
-  checkerId: 1235,
-  supportingInformation: {
-    status: CONSTANTS.DEAL_STATUS.NOT_STARTED,
-  },
-  eligibility: {
-    criteria: [
-      { id: 12, answer: null, text: 'Test' },
-    ],
-  },
-  editedBy: ['MAKER_CHECKER'],
-  submissionType: 'Automatic Inclusion Notice',
-  submissionCount: 1,
-  comments: [],
-  ukefDealId: 123456,
-  createdAt: chance.timestamp(),
-  submissionDate: chance.timestamp(),
-  portalActivities: [{
-    type: 'NOTICE',
-    timestamp: chance.timestamp(),
-    author: {
-      firstName: 'Joe',
-      lastName: 'Bloggs',
-      id: 1235,
+const MockApplicationResponseSubmitted = () => {
+  const now = new Date();
+  const yesterday = sub(now, { days: 1 });
+
+  return {
+    _id: '1234',
+    exporter: {},
+    bank: { id: 'BANKID' },
```

Comment:
Should we not also have `bankInternalRefName` inside `bank` object?

Reply:
`bankInternalRefName` is like a "name of the deal" for the bank. Could move inside bank object, but this is big change

Reply:
Okay cool can leave it for another PR